xbmc/interfaces/builtins/AndroidBuiltins.cpp

```diff
@@ -19,6 +19,8 @@
  *           params[3] = dataURI (optional)
  *           params[4] = flags (optional)
  *           params[5] = extras (optional)
+ *           params[6] = action (optional)
```

Comment:
Sorry for the delay @chrisism but adding this changes the interface for python add-ons. Please update the description on the very same file

Reply:
Ah I missed that in my last PR too.

Reply:
Do we do a new PR or continue on this one?

Reply:
Just PR the doc change separate

Reply:
And do we want a new branch for it or can I re-use the current one?
I will start a new one for a different android fix, I can also do the doc changes in that one. Let me know what is preferred.

Reply:
if you've got something else you are looking to PR, just add it as a first commit in that branch, and just comment on the PR. that'll be fine.

Reply:
Pr is merged already so you can use the same branch (after syncing with current master) or create a new one, anything will do.
Thanks and sorry for the late request